src/styles/star.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    display: flex;
+
+    &:nth-child(-n + 4) {
```

Comment:
The `:nth-child(-n + 4)` selector will not work as expected here. This selector will select the first 4 children of the parent element. But you want to select stars based on the rating. You should use a different approach. For example, you can modify your HTML structure to include both active and inactive stars, and then show the correct number of each based on the rating. Or you can use CSS classes to indicate active and inactive stars.